src/flush-microtasks.js

```diff
@@ -86,6 +87,9 @@ export default function flushMicroTasks() {
         // reproduce the problem, so there's no test for it. But it works!
         jest.advanceTimersByTime(0)
         resolve()
+      } else if (Scheduler && Scheduler.unstable_flushAll) {
+        Scheduler.unstable_flushAll()
+        enqueueTask(resolve)
       } else {
         scheduleCallback(() => enqueueTask(resolve))
       }
```

Comment:
```suggestion
      } else {
        Scheduler?.unstable_flushAll?.()
        enqueueTask(resolve)
      }
```

Reply:
I think that in that case we won't `scheduleCallback` in case we're not using the mocked scheduler, right?